tools/cockpit.spec

```diff
@@ -278,6 +308,7 @@ rm -f %{buildroot}%{_datadir}/pixmaps/cockpit-sosreport.png
 %find_lang cockpit
 %endif
 
+%if !0%{?suse_version}
```

Comment:
It seems better to move the code block that massages /usr/src/debug to here, so that everything is inside one if condition.

Reply:
will check